components/create_post/create_post.jsx

```diff
@@ -589,15 +591,17 @@ export default class CreatePost extends React.Component {
                 type = Utils.localizeMessage('create_post.post', Posts.MESSAGE_TYPES.POST);
             }
             this.props.actions.setEditingPost(lastPost.id, this.props.commentCountForPost, '#post_textbox', type);
-        } else if (!e.ctrlKey && !e.metaKey && !e.altKey && e.shiftKey && e.keyCode === KeyCodes.UP && this.state.message === '' && lastPostEl) {
+        } else if (!e.ctrlKey && !e.metaKey && !e.altKey && e.shiftKey && e.keyCode === KeyCodes.UP && this.state.message === '') {
             e.preventDefault();
-            if (document.createEvent) {
-                const evt = document.createEvent('MouseEvents');
-                evt.initMouseEvent('click', true, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
-                lastPostEl.dispatchEvent(evt);
-            } else if (document.createEventObject) {
-                const evObj = document.createEventObject();
-                lastPostEl.fireEvent('onclick', evObj);
+            const latestReplyablePostId = this.props.latestReplyablePostId;
+            const lastPost = getPost(getState(), latestReplyablePostId);
+
+            if (lastPost) {
+                AppDispatcher.handleServerAction({
```

Comment:
It would be better to put this into a redux action, at minimum can we not use AppDispatcher since that goes to the flux facade which just does a `dispatch({...action, type: ActionTypes.SELECT_POST});`

Reply:
sure thing, let me do that then

Reply:
Out of curiosity,  What was else case doing previously?

Reply:
It was simulating a click on reply icon of the last post in the channel. Definitely not the redux or react way to do things